components/input-group/examples/40-overrides.js

```diff
@@ -93,31 +98,32 @@ function Example({ brand }) {
 				}}
 			/>
 
+			<hr />
+
 			<h2>With overrides and component overrides</h2>
 			<InputGroup
+				name="example-overrides-component-1"
 				overrides={{
 					Button: {
 						styles: styles => ({
 							...styles,
-							outline: '3px dotted blue',
+							outline: '4px solid blue',
```

Comment:
Why are we making changes to the overrides tests? I don't understand in the context of a11y changes? What's the context here I'm missing?

Reply:
I just wanted to make the style overrides more obvious